systers_portal/users/forms.py

```diff
@@ -33,6 +33,8 @@ def save(self, *args, **kwargs):
 
 class SystersUserForm(forms.ModelForm):
     """Form for SystersUser model"""
+    blog_url = forms.URLField(initial="http://",widget=forms.TextInput(attrs={'placeholder': 'Enter URL'}))
+    homepage_url = forms.URLField(initial="http://",widget=forms.TextInput(attrs={'placeholder': 'Enter URL'}))
```

Comment:
You also need to adapt this in test and model.

Reply:
@tapasweni-pathak in tests the url is mentioned directly . So what to add there? Also what all to do in models.py?

Reply:
can you please check this with `makemigrations` and `migrate`.